docs/reference/agent-api.md

```diff
@@ -203,7 +235,12 @@ Before using custom context, ensure you understand the different types of [metad
 
 ## `apm.setLabel(name, value[, stringify = true])` [apm-set-label]
 
-Added in: v0.1.0<br> Renamed from `apm.setTag()` to `apm.setLabel()`: v2.10.0<br> Added `stringify` argument in: v3.11.0
+```{applies_to}
+apm_agent_node: ga 0.1.0
+```
+
+Renamed from `apm.setTag()` to `apm.setLabel()` in version 2.10.0.
+Added `stringify` argument in version 3.11.0.
```

Comment:
We could consider formatting this kind of additional information inside a note/list, but I'll let you decide if it's worth the effort. 

```suggestion
:::{admonition} Changes over time
* {applies_to}`apm_agent_node: ga 2.10.0` Renamed from `apm.setTag()` to `apm.setLabel()`.
* {applies_to}`apm_agent_node: ga 3.11.0` Added `stringify` argument.
:::
```

Reply:
I would avoid giving these notes more visibility for the time being, especially with the additional badges which may compete with the main section one...